Gruntfile.js

```diff
@@ -155,7 +155,6 @@ module.exports = function (grunt) {
             '"[MathJax]/jax/output/SVG/fonts/TeX/Main/Regular/BasicLatin.js",\n' +
             '"[MathJax]/jax/output/SVG/fonts/TeX/Main/Regular/CombDiacritMarks.js",\n' +
             '"[MathJax]/jax/output/SVG/fonts/TeX/Main/Regular/GeometricShapes.js",\n' +
-            '"[MathJax]/jax/output/SVG/fonts/TeX/Main/Regular/MathOperators.js",\n' +  // file not found
```

Comment:
Hi @pkra ,

Thanks for reviewing this PR. 
I have cleaned up files that are not available in current MathJax 2.7.1 distribution.
All is good now, this is ready to be merged.